scripts/zones/AlTaieu/mobs/Ulyovra.lua

```diff
@@ -5,24 +5,48 @@
 ---@type TMobEntity
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+end
+
 entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.REGEN, 50)
+    mob:setMobMod(xi.mobMod.WEAPON_BONUS, mob:getMainLvl() - 2) -- Base damage is level * 2
+    -- Yovra have a +50% bonus to evasion and defense.
+    mob:addMod(xi.mod.EVA, mob:getStat(xi.mod.EVA) * 0.5)
+    mob:addMod(xi.mod.DEF, mob:getStat(xi.mod.DEF) * 0.5)
```

Comment:
Isnt adding (not setting) mods on spawn dangerous?
As in, wouldn't they keep acumulating if they arent properly cleaned or something?

Reply:
From my testing, the mods are wiped between spawns. But I'll double check it again tonight.

Reply:
It looks good to me testing just now:
<img width="285" height="323" alt="image" src="https://github.com/user-attachments/assets/4062e60e-b6b7-44fe-83dd-f93d82c17631" />